package.json

```diff
@@ -33,9 +33,9 @@
     "build": "yarn build:lib && yarn build:cli",
     "build:lib": "tsc",
     "build:cli": "npx rollup -c rollup.config.js --bundleConfigAsCjs",
-    "example": "yarn --cwd example",
-    "pods": "cd example && pod-install --quiet",
-    "bootstrap": "yarn example && yarn && yarn pods"
+    "example:default": "yarn --cwd examples/default",
+    "pods": "cd examples/default && pod-install --quiet",
+    "bootstrap": "yarn example:default && yarn && yarn pods"
```

Comment:
**suggestion:** Let's keep the commands without the `:default` suffix. We can add the suffix to other non-default examples.